### PR TITLE
docs: Adding an npm policy

### PR DESCRIPTION
**Pull request by RichardLitt**
I will need sign off from everyone. :)

### CODE REVIEW

**Review by sirfumblestone — APPROVED** [review]
Looks good to me 👍

File: npm-policy.md
Comment (JaniAnttonen) [inline npm-policy.md:71]: Weren't we talking about the `orbit-chat-` naming convention? Of course we could change it later, but just as a reminder. :)

**Review by JaniAnttonen — APPROVED** [review]

**Comment by aphelionz** [comment]
:+1:

File: npm-policy.md
Comment (RichardLitt) [reply npm-policy.md:71]: Yeah! But we just archived that repository, so I figured maybe we should switch it to orbit-web? No?

**Comment by shamb0t** [comment]
Thanks for this @RichardLitt! Looks good :+1:

File: npm-policy.md
Comment (JaniAnttonen) [reply npm-policy.md:71]: I got that it was specifically with regard to the future, as in orbit-web and all that jazz could get changed to orbit-chat-[something], because Orbit is not really only about the chat, and "web" is a bit broad.

**Comment by dnoishi** [comment]
Looks great to me @RichardLitt  👍

**Review by dnoishi — APPROVED** [review]

File: npm-policy.md
Comment (haadcode) [inline npm-policy.md]: We could emphasize this with: _"Required, enable it."_

**Comment by haadcode** [comment]
Looks good to me 👍 Thanks @RichardLitt!

File: npm-policy.md
Comment (vvp) [inline npm-policy.md]: Since npm lets package owners do anything, should there be a policy or process of authorizing the user before granting them ownership?

File: npm-policy.md
Comment (vvp) [inline npm-policy.md]: Just to clarify, should the ownership to npm package be granted after the PR is merged?

Should removing a maintainer follow the same PR based process?

File: npm-policy.md
Comment (vvp) [inline npm-policy.md]: Should this be a bit more specific? Which of the 2FA modes should we enable?

**Review by vvp — COMMENTED** [review]
Looking good, just few ignorant questions :+1:

File: npm-policy.md
Comment (RichardLitt) [reply npm-policy.md]: Done.

File: npm-policy.md
Comment (RichardLitt) [reply npm-policy.md]: I was letting this one be a soft point. They must already be a maintainer of the repository. Beyond that, leave it up to the community's discretion.

**Comment by RichardLitt** [comment]
@vvp Did I answer all of your questions?

File: npm-policy.md
Comment (RichardLitt) [reply npm-policy.md]: Any of the modes, I would think. I linked to the npm article.

**Comment by vvp** [comment]
@RichardLitt you did :slightly_smiling_face: LGTM now

**Review by vvp — APPROVED** [review]

**Pull request merged**
